src/helpers/localStorageHelpers.js

```diff
@@ -81,17 +81,16 @@ export const removeFavoriteCategory = (category) => {
   return true;
 };
 
-export const getDrafts = () => {
-  return store.get('drafts') || {};
-};
+export const getLayout = () =>
+  store.get('layout') || 'card';
```

Comment:
I'm against this whole idea of using `store` package to directly work with localStorage. The reason is that we are changing the state and then change the localStorage and it makes a duplication of source of truth and makes it prone to errors in the logic. That's why I used persistence methods for `favorites` feature like here:

https://github.com/adcpm/busy/blob/dev/src/store.js#L63

Just make sure the redux state is saving the values in a separate key and add it to the line above like `favorites` and it stay persistent without storing and reloading manually.